libs/langchain/tests/unit_tests/document_loaders/parsers/language/test_cobol.py

```diff
@@ -0,0 +1,42 @@
+import unittest
```

Comment:
well - if you see in test_javascript.py and test_python.py they use unittest
I think i ought to follow the pattern instead of trying to change anything

Reply:
I basically just copied test_python.py and changed it to cobol